spec/rails_helper.rb

```diff
@@ -29,6 +29,15 @@
   puts e.to_s.strip
   exit 1
 end
+
+RSpec.shared_context 'Authenticated User' do
+  let(:user) { create(:user) }
```

Comment:
let's put this in a separate file, so it's clearly visible and refactorable.
For example, in spec/support/shared_contexts/authenticated_user.rb